docs/examples.md

```diff
@@ -0,0 +1,39 @@
+**Selecting the first match via a custom Swift matcher**
+
+When an application has exact duplicate elements (every attribute is the same,
+including their hierarchy), then the correct fix is to update the app to avoid
+duplicating the UI. When that's not possible, a workaround is to match on
+the first element.
+
+```swift
+// Swift Custom Matcher
+/*
+ *  Example Usage:
+ *
+ *  EarlGrey().selectElementWithMatcher(grey_allOfMatchers(
+ *    grey_accessibilityID("some_id"),
+ *    grey_interactable(),
+ *    grey_firstElement()))
```

Comment:
worth mentioning that order of matcher is important (most -> least selective) and grey_firstElement should come last since it is the least selective matcher.